sosw/components/helpers.py

```diff
@@ -892,6 +896,13 @@ def to_bool(val):
     raise Exception(f"Can't convert unexpected value to bool: {val}, type: {type(val)}")
 
 
+def _unwrap_msg_dict_from_sns_event(event) -> Dict:
+    try:
+        return json.loads(event['Records'][0]['Sns']['Message'])
```

Comment:
I suggest to add a validation meanwhile that there is ONLY ONE record in the event.
At some point we shall face some batches of SNS messages coming in same event to the Lambda. We better notice this immediately.

Reply:
added